src/utils/blockchain.utils.tsx

```
@@ -2,22 +2,25 @@ import * as web3 from "@solana/web3.js";
 import * as splToken from "@solana/spl-token";
 import { SolanaWalletsEnum } from "@/models";
 import { PublicKey } from "@solana/web3.js";
+import { SolanaWalletProvider } from "@/models/app.model";
 
-export const getSolanaWalletsProvider = (solWallet?: SolanaWalletsEnum) => {
-  if (typeof window === undefined) return undefined;
+export const getSolanaWalletsProvider = (
+  solWallet?: SolanaWalletsEnum
+): SolanaWalletProvider | null => {
+  if (typeof window === undefined) return null;
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)